pkg/network/network.go

```diff
@@ -206,38 +218,47 @@ func defaultConfig() *Config {
 
 // NewConfigFromConfigMap creates a Config from the supplied ConfigMap
 func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
+	return NewConfigFromMap(configMap.Data, configMap.Name)
+}
+
+// NewConfigFromMap creates a Config from the supplied data.
+func NewConfigFromMap(data map[string]string, cmName string) (*Config, error) {
```

Comment:
Why would we pass the cmName in here? Feels like a weird interface and is it not consistent with the other occurrences of it.

Reply:
Stupid logging, I can just constant it out.

Reply:
Removed.